phpunit.xml.dist

```diff
@@ -53,7 +53,7 @@
 
     <php>
         <ini name="zend.enable_gc" value="0"/>
-        <ini name="memory_limit" value="10G"/>
+        <ini name="memory_limit" value="-1"/>
```

Comment:
Why do it with half-measures if we can have it directly in PHP config?

Reply:
Docker or `phpunit.xml`?

Reply:
i decided to merge this PR as is.